internal/controllers/rollout/controller.go

```diff
@@ -58,12 +58,6 @@ func (c *controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	var latestRollout time.Time
 	for _, comp := range compList.Items {
-		// For every synthesizer, only one composition can be the target of a rolling change at any point in time.
-		// To avoid blocked rollouts caused by compositions that cannot be synthesized, also time out and move on eventually.
-		if isRolling(&comp) {
```

Comment:
This was always just a perf optimization but removing it isn't a huge deal. Better than risk more race conditions 😬